components/ActivityForm/index.js

```diff
@@ -23,19 +21,15 @@ export default function ActivityForm({ onSubmit, initialData, isEditMode }) {
 
   return (
     <>
-      <StyledBackLink href={isEditMode ? `/${initialData?.id}` : "/"}>
-        <Icon name="chevronLeft" />
-        {isEditMode ? "Discard changes" : "Back to all Activities"}
-      </StyledBackLink>
       <StyledForm onSubmit={handleSubmit}>
         <StyledLabel htmlFor="title">Activity</StyledLabel>
         <StyledInput
           id="title"
           name="title"
           type="text"
-          maxLength="150"
           placeholder="add activity-title"
           defaultValue={initialData?.title}
+          pattern="[A-Za-z][A-Za-z\s]{2,35}"
```

Comment:
Nice idea but what if the title has two words?

Reply:
It stilll works with two or more word, since we used "/s"